src/UI/TransactionsToastList/components/TransactionToast/types.ts

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { TxDetailsProps } from 'UI/TxDetails';
+import { ProgressProps } from 'UI/Progress';
+import { TransactionToastContentProps } from './TransactionToastContent';
+import { SignedTransactionType, TransactionBatchStatusesEnum } from 'types';
+import { FontAwesomeIconProps } from '@fortawesome/react-fontawesome';
+
+export type TransactionToastDefaultProps = {
+  toastId: string;
+  transactions?: SignedTransactionType[];
+  status?: TransactionBatchStatusesEnum;
+  lifetimeAfterSuccess?: number;
+  endTimeProgress?: number;
+  startTimestamp?: number;
+  className?: string;
+  shouldRenderDefaultCss?: boolean;
+  onDelete?: (toastId: string) => void;
+};
+
+export type TransactionToastContentElementsProps = {
+  TransactionToastStatusIcon?: React.ComponentType<
```

Comment:
It seems like the type is being copy-pasted multiple times. I would create a new type with a generic attribute instead